node/node.go

```diff
@@ -313,6 +319,7 @@ func New(host p2p.Host, consensus *bft.Consensus, db ethdb.Database) *Node {
 
 	if consensus != nil && consensus.IsLeader {
 		node.State = NodeLeader
+		go node.ReceiveClientGroupMessage()
```

Comment:
try to move this go routine and the go node.ReceiveGroupMessage() in to node_handler.go because they are related to networking

Reply:
sure. will look into that.